front/app/components/admin/PostManager/components/FilterSidebar/statuses/ScreeningStatusFilter.tsx

```diff
@@ -86,15 +115,20 @@ const ScreeningStatusFilter = ({ status, active, onClick }: Props) => {
                 <StatusText>
                   <T value={status.attributes.title_multiloc} />
                 </StatusText>
-                <IconTooltip
-                  theme="light"
-                  iconSize="16px"
-                  content={
-                    <FormattedMessage
-                      {...messages.automatedStatusTooltipText}
-                    />
-                  }
-                />
+                {statusFilterIsEnabled && (
```

Comment:
This tooltip would show up underneath other tooltips while the filter was disabled.